src/components/ToolButton.tsx

```diff
@@ -33,6 +33,8 @@ interface ToolButtonProps {
   disabled?: boolean;
   tooltipText?: string;
   className?: string;
+  toggle?: boolean;
+  customSx?: boolean;
```

Comment:
I'd prefer if we just name it `sx`:

```suggestion
  sx?: boolean;
```

Reply:
Sorry  its mistake I forgot to remove it, not using it anymore. Will just remove it